app/clients/hedera/mirror-node/client.go

```diff
@@ -14,577 +14,578 @@
  * limitations under the License.
  */
 
-package mirror_node
```

Comment:
What are the changes in this file? Is it code formatting or it is related to the updates in the PR?

Reply:
I am sorry about this. It seems like VScode automatically added a "space" in front of the whole code from line 17 to the end. Maybe it was trying to align the code with the end of the comment on line 15 " */"

Reply:
Removed it